integrations/metabase/nango.yaml

```
@@ -0,0 +1,70 @@
+integrations:
+    metabase:
+        actions:
+            create-user:
+                description: Creates a user in Metabase.
+                output: User
+                endpoint:
+                    method: POST
+                    path: /users
+                    group: Users
+                input: MetabaseCreateUser
+            delete-user:
+                description: Deletes a user in Metabase by id.
+                endpoint:
+                    method: DELETE
+                    path: /users
+                    group: Users
+                output: SuccessResponse
+                input: MetabaseDeleteUser
```

```suggestion
                input: IdEntity
```

- delete `MetabaseDeleteUser`